vnext/ReactUWP/Modules/TimingModule.cpp

```diff
@@ -90,8 +90,11 @@ std::weak_ptr<facebook::react::Instance> Timing::getInstance() noexcept {
 }
 
 void Timing::OnRendering(
+#ifndef HEADLESS_JS
```

Comment:
>HEADLESS_JS [](start = 8, length = 11)

Do we need to use feature flag or things to control it? A macro is hard to test and maintain

Reply:
We can probably do without the macro. I initially thought it would be nice to use to prevent these changes regressing other consumers (in case of bugs). But we have been consuming this code in our apps for a month or so now without issues. 

I do need to revisit these changes to make them better overall. (These changes don't work correctly when there can be multiple views.) but now i have higher confidence in what the problematic areas are and plan to update this soon